Source/SourceKittenFramework/library_wrapper.swift

```diff
@@ -1,16 +1,62 @@
 import Foundation
+#if os(Windows)
+import WinSDK
+
+@_transparent
+internal func MAKELANGID(_ p: WORD, _ s: WORD) -> DWORD {
+    return DWORD((s << 10) | p)
+}
+
+struct WindowsError {
```

Comment:
Can you please move this to its own file? `WindowsError.swift`

Reply:
Should this conform to `Error`?

Reply:
Sure, can move the `WindowsError` into its own file.  Conforming to `Error` seems like a good idea.

Reply:
Done with #784